File: Juniper/217.存在重复元素.py
#
# @lc app=leetcode.cn id=217 lang=python3
#
# [217] 存在重复元素
#
class Solution:
    def containsDuplicate(self, nums: List[int]) -> bool:
        dic = dict()
        for num in nums:
            if num in dic:
                return True
            dic[num] = 1
        return False
      
            
        

File: hahlw/腾讯50题/217. 存在重复元素.cpp
#include<iostream>
#include<unordered_map>
#include<vector>
using namespace std;
class Solution {
public:
    bool containsDuplicate(vector<int>& nums) {
        unordered_map<int,int>mp;
        for(auto n :nums)
        {
            if(mp.find(n)!=mp.end())return true;
            else{
                mp[n]=1;
            }
        }
        return false;
    }
};